src/CharCreation/Attributes.elm

```diff
@@ -89,6 +92,17 @@ progressBarStyle val =
         , ( "min-width", "0" )
         ]
 
+tickStyle: Int -> Html.Attribute Msg
+tickStyle val =
+    style
+        [ ( "width", (toString val) ++ "%" )
```

Comment:
Nitpick: `++` has lower precedence than function application, so we can remove the parens around `toString val` so that it becomes:

`[ ( "width", toString val ++ "%" )`